Documentation/ApiOverview/ContentSecurityPolicy/Index.rst

```diff
@@ -192,9 +192,27 @@ is available, which provides the nonce in a Fluid template, for example:
 
 ..  code-block:: html
 
-    <script nonce="{f:security.nonce()}">const inline = 'script';</script>
-    <f:asset.script useNonce="1">console.log('Something');</f:asset.script>
+    <script nonce="{f:security.nonce()}">
```

Comment:
can we use literalincludes for these as they are easier to format and adjust? Also a caption would be helpful which also demonstrates that this is fluid, not plain html.

Same some lines bellow

Reply:
I don't see the need for an external file for this small examples. If you think it is necessary, you can move them into separate files in a follow-up.

And feel free to add a caption as suggestion.